package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Visualise your DAG layouts and chunk replication",
   "main": "src/index.js",
   "dependencies": {
+    "@ipfs-shipyard/ignite-metrics": "^1.2.1",
```

Comment:
did you want latest here? others are using 1.2.2

Reply:
updated